### PR TITLE
Minor modifications and error fixing.

### PR DESCRIPTION
**Pull request by pbeckm**
* Commented out references to Diffie-Hellman parameter files.
* Fixed path to nginx ssl-config file.
* Using bare asterisk (*) for (virtual) server name in nginx is
  prohibited and thus misleading in a template file.

### CODE REVIEW

**Comment by schaarj** [comment]
two suggestions before merging:
* replace `*` in server name with `_` which should work if I'm not mistaken or  with `[URL]` and add a command in the documentation to replace `[URL]` with the actual server name as done for the other services
*  instead of not using Diffie-Hellman by default just add the command to generate the parameters to the documentation (`openssl dhparam -out dhparams.pem 4096`)

**Comment by pbeckm** [comment]
ok with both points, then I use  [URL] here as well for consistency with other occurrences.  Using the quite verbose sed-stuff adds to confusion, though. I'd suggest a sentence like "Next, edit the files foo and bar to fit Your requirements" in the documentation.    
Also, some guiding comments in the files themselves won't hurt! 

I will merge main into my branch. Then push the changes, possibly with some minor alterations on the docs.

**Comment by pbeckm** [comment]
I wasn't aware that there are multiple occurrences  of [URL] in the files, hence sed's a time-saver, until we find a better solution, though.  I updated the section on manually retrieved certs accordingly.

File: compose-templates/docker-compose-nginx-ssl.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ services:
     secrets:
       - cert.pem
       - key.pem
+      # Generate this file with the command: "openssl dhparam -out dhparam.pem 4096"
```
Comment (schaarj) [inline compose-templates/docker-compose-nginx-ssl.yml:15]: I would add this to the deployment guide explicitly as well

Comment (pbeckm) [reply compose-templates/docker-compose-nginx-ssl.yml:15]: Yes, please!

**Pull request merged**